test/runtests.jl

```diff
@@ -54,6 +54,8 @@ end
 @info "sleeping for 60 seconds to ensure Azure clean-up from any previous run"
 sleep(60)
 
+@ccall AzStorage.libAzStorage.resetPerfCounters()::Cvoid
```

Comment:
We probably want a trivial julia wrapper method for "reset_perf_counters" rather than directly calling the c function, and same for "get_perf_counters".